uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -40,40 +39,41 @@ Feature: Greengrass V2 LogManager
         """
         And I deploy the Greengrass deployment configuration
         Then the Greengrass deployment is COMPLETED on the device after 2 minutes
-        Then I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
+        And I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
+        And I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
         And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
-        And I verify the rotated files are not deleted except for the active log file for component UserComponentA
 
-    @smoke
-    Scenario: LogManager-1-T2: As a customer I can configure the logs uploader to delete log files after all logs from the file have been uploaded to CloudWatch
```

Comment:
Now you removed the `LogManager-1-T2: `. Main currently has 2 tests. Adding this one it should have 3. You are still deleting one